src/common/Utilities/SFMTRand.cpp

```diff
@@ -24,7 +24,7 @@
 
 #if __has_include(<mm_malloc.h>)
 #include <mm_malloc.h>
-#elif __has_include(<malloc.h>)
+#elif __has_include(<malloc.h>) && (!defined(__aarch64__) && TRINITY_COMPILER == TRINITY_COMPILER_GNU)
```

Comment:
This one is for msvc only, just replace your extra condtion with TRINITY_COMPILER == TRINITY_COMPILER_MICROSOFT